packages/lumx-react/src/components/thumbnail/Thumbnail.tsx

```diff
@@ -158,50 +159,65 @@ const Thumbnail: React.FC<ThumbnailProps> = ({
     imgProps,
     ...props
 }: ThumbnailProps): ReactElement => {
-    const focusImageRef = useFocusedImage(focusPoint!, aspectRatio!, size!, debounceTime!, isFollowingWindowSize!);
-
+    const { isLoaded, hasError } = useImage(image);
+    const focusImageRef = useFocusedImage(
+        focusPoint!,
+        aspectRatio!,
+        size!,
+        resizeDebounceTime!,
+        isFollowingWindowSize!,
+        isLoaded,
+    );
     const setCrossOrigin = () => {
         return !isInternetExplorer() && isCrossOriginEnabled ? crossOrigin : undefined;
     };
 
-    return (
-        <div
-            className={classNames(
-                className,
-                handleBasicClasses({ align, aspectRatio, prefix: CLASSNAME, size, theme, variant }),
-                {
-                    [`${CLASSNAME}--fill-height`]: fillHeight,
-                },
-            )}
-            tabIndex={isFunction(onClick) ? 0 : -1}
-            onClick={onClick}
-            onKeyDown={onEnterPressed(onClick)}
-            {...props}
-        >
-            {aspectRatio === AspectRatio.original ? (
-                <img
-                    {...(imgProps || {})}
-                    ref={focusImageRef}
-                    className={`${CLASSNAME}__image`}
-                    src={image}
-                    alt={alt}
-                    loading={loading}
-                />
-            ) : (
-                <div className={`${CLASSNAME}__background`}>
+    if (hasError) {
+        return <span>Image broken</span>;
```

Comment:
Should not we display translated text or icon?

Reply:
Still WIP :)